requirements.txt

```diff
@@ -0,0 +1,112 @@
+black 23.1.0 The uncompromising code formatter.
+├── click >=8.0.0
+│   └── colorama * 
+├── mypy-extensions >=0.4.3
+├── packaging >=22.0
+├── pathspec >=0.9.0
+├── platformdirs >=2
+└── tomli >=1.1.0
+fastapi 0.89.1 FastAPI framework, high performance, easy to learn, fast to code, ready for production
+├── pydantic >=1.6.2,<1.7 || >1.7,<1.7.1 || >1.7.1,<1.7.2 || >1.7.2,<1.7.3 || >1.7.3,<1.8 || >1.8,<1.8.1 || >1.8.1,<2.0.0
+│   └── typing-extensions >=4.2.0 
+└── starlette 0.22.0
+    └── anyio >=3.4.0,<5 
+        ├── idna >=2.8 
+        └── sniffio >=1.1 
+httpx 0.23.3 The next generation HTTP client.
+├── certifi *
+├── httpcore >=0.15.0,<0.17.0
+│   ├── anyio >=3.0,<5.0 
+│   │   ├── idna >=2.8 
+│   │   └── sniffio >=1.1 
+│   ├── certifi * 
+│   ├── h11 >=0.13,<0.15 
+│   └── sniffio >=1.0.0,<2.0.0 (circular dependency aborted here)
+├── rfc3986 >=1.3,<2
+│   └── idna * 
+└── sniffio *
+ipykernel 6.21.1 IPython Kernel for Jupyter
+├── appnope *
+├── comm >=0.1.1
+│   └── traitlets >=5.3 
+├── debugpy >=1.6.5
+├── ipython >=7.23.1
+│   ├── appnope * 
+│   ├── backcall * 
+│   ├── colorama * 
+│   ├── decorator * 
+│   ├── jedi >=0.16 
+│   │   └── parso >=0.8.0,<0.9.0 
+│   ├── matplotlib-inline * 
+│   │   └── traitlets * 
+│   ├── pexpect >4.3 
+│   │   └── ptyprocess >=0.5 
+│   ├── pickleshare * 
+│   ├── prompt-toolkit >=3.0.30,<3.1.0 
+│   │   └── wcwidth * 
+│   ├── pygments >=2.4.0 
+│   ├── stack-data * 
+│   │   ├── asttokens >=2.1.0 
+│   │   │   └── six * 
+│   │   ├── executing >=1.2.0 
+│   │   └── pure-eval * 
+│   └── traitlets >=5 (circular dependency aborted here)
+├── jupyter-client >=6.1.12
+│   ├── jupyter-core >=4.12,<5.0.0 || >=5.1.0 
+│   │   ├── platformdirs >=2.5 
+│   │   ├── pywin32 >=1.0 
+│   │   └── traitlets >=5.3 
+│   ├── python-dateutil >=2.8.2 
+│   │   └── six >=1.5 
+│   ├── pyzmq >=23.0 
+│   │   └── cffi * 
+│   │       └── pycparser * 
+│   ├── tornado >=6.2 
+│   └── traitlets >=5.3 (circular dependency aborted here)
+├── jupyter-core >=4.12,<5.0.0 || >=5.1.0
+│   ├── platformdirs >=2.5 
+│   ├── pywin32 >=1.0 
+│   └── traitlets >=5.3 
+├── matplotlib-inline >=0.1
+│   └── traitlets * 
+├── nest-asyncio *
+├── packaging *
+├── psutil *
+├── pyzmq >=17
+│   └── cffi * 
+│       └── pycparser * 
+├── tornado >=6.1
+└── traitlets >=5.4.0
+mypy 0.991 Optional static typing for Python
```

Comment:
We don't need packages in pyproject.toml's dev group like formatters, linters to take space in docker container for production. Can we avoid them here? Feel free to turn it to issue if you dont want to tackle it now.

Reply:
made it an issue! 💯